.circleci/config.yml

```diff
@@ -112,7 +130,9 @@ workflows:
     jobs:
       - build-deploy:
           requires:
-            - test-nightly
+            - test-nightly-firefox
             - test-release-firefox
-      - test-nightly
-      - test-release-firefox
\ No newline at end of file
+            - test-beta-firefox
+      - test-nightly-firefox
+      - test-release-firefox
+      - test-beta-firefox
```

Comment:
Maybe I'm missin' somethin'&hellip; Does this need to run these jobs, or are they already triggered by the `build-deploy`'s `requires:` above on L133-135?

Reply:
You have to list the jobs that run per PR